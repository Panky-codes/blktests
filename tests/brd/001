#!/bin/bash
# SPDX-License-Identifier: GPL-3.0+
# Copyright (C) 2023 Pankaj Raghav <p.raghav@samsung.com>
# A simple fio test to check persistence

. tests/brd/rc
DESCRIPTION="run a fio workload with verify to check persistence"

QUICK=1

test() {
	echo "Running ${TEST_NAME}"

	modprobe brd rd_size=102400 rd_nr=1

	_fio_perf --bs=4k --rw=write --name=verifywrites \
		--filename=/dev/ram0 --size=100M \
		--verify=crc32 --ioengine=io_uring

	modprobe -r brd

	echo "Test complete"
}
