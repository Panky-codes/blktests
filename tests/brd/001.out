Running brd/001
Test complete
